hive-interface.js

```diff
@@ -418,11 +428,53 @@ class Hive {
 	}
 
 	async processTxQueue() {
-		if(this.tx_queue.length <= 0) return;
+		this.clear_ctr++;
+		// Every 3 seconds, or one block, clear the transaction counts
+		if(this.clear_ctr >= 3)
+		{
+			for (const key of this.accounts_used.keys()) {
+			this.accounts_used.set(key, 0);
+			}
+			this.clear_ctr = 0;
+		}
+		// If the queue is empty, exit here
+		if(this.tx_queue.length <= 0) return;		
+		let exit = false;
+		let next_account = this.tx_queue[0].data.required_auths.length > 0 ? this.tx_queue[0].data.required_auths[0] : this.tx_queue[0].data.required_posting_auths[0];
+		if(this.accounts_used.has(next_account)) {
```

Comment:
since we reuse this functionality in multiple places you may want to create method for it.